docs/page.js

```diff
@@ -23,6 +23,56 @@ var onDocumentLoad = function ( event ) {
 			break;
 
 	}
+    
+
+    // redirect, if the page is not yet displayed as an iframe (with navigation panel and working links)    
+    
+    if ( !window.frameElement ) {
+        
+        var redirectUrl;
+
+        for ( var mainCategory in list ) {
+            
+            var subCategories = list[ mainCategory ];
+            
+            for ( var subCategory in subCategories ) {
+                
+                var pages = subCategories[ subCategory ]
+                
+                for ( i = 0; i < pages.length; i++ ) {
+                    
+                    var page = pages[ i ];
+
+                    if ( page[ 1 ] === section + '/' + path ) {  
+
+                        redirectUrl = 'index.html#' 
```

Comment:
Of course `index.html` is redundant. Seems that I didn’t see the forest for the trees ;-)

Thanks!